prow/plugins/plugins.go

```diff
@@ -222,6 +222,12 @@ type Owners struct {
 	// the approve and lgtm plugins to use solely OWNERS files for access
 	// control in the provided repos.
 	SkipCollaborators []string `json:"skip_collaborators,omitempty"`
+
+	// LabelsBlackList holds a list of labels that should not be present in
+	// the root directory OWNERS file, to prevent their automatic addition
```

Comment:
This isn't correct, the labels shouldn't be present in any OWNERS file, not just the root dir OWNERS file.